src/scripts/main.js

```diff
@@ -1,3 +1,48 @@
 'use strict';
 
-// write your code here
+let bodyDoc;
+let mateLogo;
+
+if (document.querySelector('body') !== null) {
+  bodyDoc = document.querySelector('body');
```

Comment:
You are querying the DOM for the 'body' element twice. Consider storing the result of 'document.querySelector('body')' in a variable outside of the if-statement and then check if that variable is not null to avoid unnecessary DOM queries.